app/models/user.rb

```diff
@@ -1,7 +1,11 @@
 class User < ApplicationRecord
-  has_many :comments, foreign_key: :author_id
-  has_many :posts, foreign_key: :author_id
-  has_many :likes, foreign_key: :author_id
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
As per project requirements kindly make sure that user registration asks for email confirmation.

![image](https://github.com/MarkoKermi/Blog_app/assets/38088818/8d05efaf-d103-46eb-926b-01922b91ffb8)

- You can achieve this by uncommenting ` # :confirmable` in the User model.

Reply:
@KDlamini thx for the suggestion and review!